tt1-blocks/readme.txt

```diff
@@ -1,9 +1,9 @@
 === TT1 Blocks ===
 Contributors: wordpressdotorg
 Requires at least: 5.6
-Tested up to: 5.7
+Tested up to: 5.8
```

Comment:
Should this be 5.9?

Reply:
5.9 is still beta. I think we need to update it when 5.9 is available.